packages/dashboard/lib/DashboardServer.ts

```diff
@@ -95,6 +98,62 @@ export class DashboardServer {
       this.expressApp.post("/rpc", this.postRpc.bind(this));
     }
 
+    this.expressApp.get("/fetch-and-compile", async (req, res) => {
+      const { address, networkId, etherscanApiKey } = req.query as Record<
+        string,
+        string
+      >;
+      let config;
+      try {
+        config = Config.detect();
+      } catch (error) {
+        const notFound = "Could not find suitable configuration file.";
+        if (!error.message.includes(notFound)) {
+          throw error;
+        }
+      }
```

Comment:
I don't think this should throw? Since this config isn't actually used for fetch and compile, only to grab the etherscan key if it's there.

Reply:
Hmm, I don't know. The point here is the same with the other `try/catch`. If we can't find a config, we want to move on and not care. However, if it throws in some unexpected way we want to know about it. In this case though it might not matter...maybe we should just leave the `catch` empty? Do we want to know when it fails for some other reason than not being able to find it?

Reply:
Do nothing/log, and then move on to fetch and compile seems reasonable to me. But I understand your point with this and the try/catch block below though. If in react we also wrap the request in try/catch then my concerns are not  important here.

Reply:
Ok, in this case I think it makes sense to just ignore errors with `Config.detect` - the other one below, instead of throwing should probably 500 like you said